ltr/concourse/pipeline.yaml

```diff
@@ -75,6 +75,13 @@ resources:
     bucket: ((readonly_private_bucket_name))
     region_name: eu-west-2
     regexp: search-learn-to-rank/integration-model-(.*).txt
+# AB Test variant B (list_mle_model)
+- name: integration-model-variant-b
+  type: s3-iam
+  source:
+    bucket: ((readonly_private_bucket_name))
+    region_name: eu-west-2
+    regexp: search-learn-to-rank/integration-model-variant-b-([0-9]*).txt
```

Comment:
I think the other regexes will need to change too (or at least the `search-learn-to-rank/integration-model-(.*).txt` one)